cmdeploy/src/cmdeploy/service/chatmail-metadata.service.f

```diff
@@ -5,6 +5,41 @@
 ExecStart={execpath} /run/dovecot/metadata.socket vmail /home/vmail/mail/{mail_domain}
 Restart=always
 RestartSec=30
+User=vmail
+CapabilityBoundingSet=
```

Comment:
please add a comment here with the invocation that created these auto-generated security settings (i presume they are). 
Also i wonder if these security-settings shouldn't be done in a separate PR applied to all services?